jerry-core/jcontext/jcontext.h

```diff
@@ -265,6 +272,7 @@ extern jerry_hash_table_t jerry_global_hash_table;
 /**
  * Provides a reference to a field in the current context.
  */
+#define JERRY_INIT_CURRENT_INSTANCE()
```

Comment:
- Doc: injecting this macro here "steals" the documentation from `JERRY_CONTEXT`. E.g., the new macro should be placed _before_ the doc of `JERRY_CONTEXT` (and should have its own documentation, e.g., something like the one which was misplaced around line 147).